changelog/911.breaking.rst

```diff
@@ -0,0 +1,5 @@
+Removed default Particle types for functions in `formulary/parameters.py`. Any
+code which had previously used the default value for these arguments will need
+to be updated by passing the appropriate particle type as a parameter.
+
+Documentation and tests for these functions have been updated as needed.
```

Comment:
```suggestion
Removed default values for the `ion` and `particle` arguments of functions contained in `plasmapy.formulary.parameters`, in accordance with issue [#453](https://github.com/PlasmaPy/PlasmaPy/issues/453).
```

You want to keep this as one paragraph because it gets translated to a single bullet point in a reST unordered list.